dpnp/dpnp_array.py

```diff
@@ -1115,7 +1115,38 @@ def nbytes(self):
 
     @property
     def ndim(self):
-        """Number of array dimensions."""
+        """
+        Return the number of dimensions of an array.
+
+        For full documentation refer to :obj:`numpy.ndarray.ndim`.
```

Comment:
Do we need to refer to :obj:`dpnp.ndim` like for other function?

Reply:
I did not refer to `dpnp.ndim` because they are slightly different. `dpnp.ndim` works for array_like input but `dpnp.ndarray.ndim` only works for `dpnp.ndarray` or `usm_ndarray`. This is kind of comparable to `dpnp.shape` and `dpnp.ndarray.shape`.